ci.adoc

```diff
@@ -12,12 +12,19 @@ when referencing nodes, e.g. in the `Jenkinsfile`.
 
 * `linux` : A Linux (Ubuntu 18.04 LTS) instance (alias of `java`)
 * `docker` : A Linux (Ubuntu 18.04 LTS) instance with a running Docker daemon
-* `windows` : A "stock" Windows 2012 R2 provisioned on Azure
-(link:https://azure.microsoft.com/en-us/documentation/articles/cloud-services-sizes-specs/[Standard D3 v2])
+* `windows` : A Windows 2019 provisioned on Azure or AWS
 * `highmem` : A Linux (Ubuntu 18.04 LTS) instance with 4vCPU/28GB RAM
 (link:https://azure.microsoft.com/en-us/documentation/articles/cloud-services-sizes-specs/[Standard A6]). Please avoid unless running ATH or other high-memory capacity instances.
+* `windock` : A Windows 2019 provisioned on Azure or AWS that is able to run Windows Docker images
 * `puppet` : A Linux (Ubuntu 14.04 LTS) instance that is managed by link:https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/manifests/buildslave.pp[this Puppet code]
 
+==== Node Labels - Processors
+
+* `amd64` : An agent running an AMD or Intel 64 bit processor
+* `arm64` : An agent running an ARM 64 bit processor on AWS (Amazon Graviton)
+* `ppc64le` : An agent running an IBM PowerPC 64 bit processor
+* `s390x` : An agent running an IBM System 390x (Z series) processor
```

Comment:
Is this really a thing?

Reply:
> Is this really a thing?

Yes, it is.  IBM has donated access to an IBM System 390x running Ubuntu Linux 18.04 and it is attached to ci.jenkins.io as https://ci.jenkins.io/computer/s390x-agent/systemInfo.  Output looks like this:


Name       | Value 
-- | --
java.library.path | /usr/java/packages/lib:/usr/lib/s390x-linux-gnu/jni:/lib/s390x-linux-gnu:/usr/lib/s390x-linux-gnu:/usr/lib/jni:/lib:/usr/lib
java.runtime.name | OpenJDK Runtime Environment
java.runtime.version | 11.0.7+10-post-Ubuntu-2ubuntu218.04
os.arch | s390x
os.name | Linux
os.version | 4.15.0-99-generic

Runs very nicely.

Reply:
:exploding_head: